### PR TITLE
Product update actions

### PR DESCRIPTION
**Pull request by schleichardt**
@lauraluiz please review.

Release notes are in http://sphereio.github.io/sphere-jvm-sdk/javadoc/product-update-actions/io/sphere/sdk/meta/ReleaseNotes.html

### CODE REVIEW

File: common/src/main/java/io/sphere/sdk/models/LocalizedStrings.java [hunk 1]
```diff
@@ -137,24 +137,10 @@ public String toString() {
...
-        if (o == null || getClass() != o.getClass()) return false;
-        LocalizedStrings localized = (LocalizedStrings) o;
-        return this.translations.equals(localized.translations);
-    }
-
-    @Override
-    public int hashCode() {
-        return translations.hashCode();
-    }
-
```
Comment (lauraluiz) [inline common/src/main/java/io/sphere/sdk/models/LocalizedStrings.java]: Why are the equality methods not good here?

Comment (schleichardt) [inline common/src/main/java/io/sphere/sdk/models/LocalizedStrings.java]: The class was mostly from the Play SDK, in the JVM SDK we just extend Base and do not reimplement equals, hashCode and toString.

Comment (lauraluiz) [inline common/src/main/java/io/sphere/sdk/models/LocalizedStrings.java]: Oops true, I just forgot we had `Base` (as per usual) :D

**Comment by lauraluiz** [comment]
Looks good!

**Pull request merged**